catalog/systemd.catalog.in

```diff
@@ -119,6 +119,16 @@ Documentation: sd-login(3)
 
 A seat @SEAT_ID@ has been removed and is no longer available.
 
+-- b2bcbaf5edf948e093ce50bbea0e81ec
+Subject: The Secure Attention Key (SAK) pressed on @SEAT_ID@
+Defined-By: systemd
+Support: %SUPPORT_URL%
+Documentation: sd-login(3)
```

Comment:
that's a link to the man page of the C API of systemd-logind.

Please use `man:systemd-logind.service(8)` here instead.